src/ralph/models/xapi/video/fields/verbs.py

```diff
@@ -0,0 +1,19 @@
+"""Video xAPI events verb fields definitions"""
+
+from ...base import BaseModelWithConfig
+from ...constants import LANG_EN_DISPLAY, VERB_PLAYED_DISPLAY
+from ..constants import VERB_VIDEO_PLAYED_ID
+
+
+class VideoPlayedVerbField(BaseModelWithConfig):
+    """Represents the `verb` field for video played xAPI statement.
+
+    Attributes:
+        id (str): Consists of the value `https://w3id.org/xapi/video/verbs/played`.
+        display (dict): Consists of the dictionary `{"en": "played"}`.
+    """
+
+    id: VERB_VIDEO_PLAYED_ID = VERB_VIDEO_PLAYED_ID.__args__[0]
+    display: dict[LANG_EN_DISPLAY, VERB_PLAYED_DISPLAY] = {
```

Comment:
It seems Marsha uses 'en-US' instead of 'en' for the display name ?

Reply:
That's right, I did not pay attention to change it. Thanks for noticing it!